DynamicMvvm.sln

```diff
@@ -56,6 +56,7 @@ Global
 		{FB040CF7-8A52-412E-964F-B8BE3B025746}.Release|UAP.ActiveCfg = Release|Any CPU
 		{FB040CF7-8A52-412E-964F-B8BE3B025746}.Release|UAP.Build.0 = Release|Any CPU
 		{10AE5B44-4233-4AD9-BF71-FEDBBC4B2C29}.Debug|Android.ActiveCfg = Debug|Any CPU
+		{10AE5B44-4233-4AD9-BF71-FEDBBC4B2C29}.Debug|Android.Build.0 = Debug|Any CPU
```

Comment:
Is that necessary? :)

Reply:
well, I wanted to be able to build/run tests, but looking at the change log, it's weird that it says android. Let me re validate that

Reply:
well I had issues, because my visual studio was set to build on Android. I've reverted this changes. Good catch!